bbc/simorgh
# `av-embeds` - AMP Articles to use `parentPID` instead of `versionPID`

Resolves JIRA [number]

Overall changes
======
- Amends the AMP and `externalEmbedUrl` URLs for Optimo Articles av-embeds to use the `parentPID` value, rather than `versionPID` when requesting a media asset. The BFF can handle either the `parentPID` or `versionPID`, and in these cases using the `parentPID` should somewhat help resolve requests for media assets that no longer exist

Code changes
======
- Passes new `parentPID` param into AMP and external URL constructors, using this value for Optimo Articles only

Testing
======
1. Update this line to be `https://web-cdn.api.bbci.co.uk` - https://github.com/bbc/simorgh/blob/cbee84c9bbca7eab30c898d795a4e69f901065ac/src/app/components/MediaLoader/utils/urlConstructors.ts#L8
2. Visit http://localhost.bbc.com:7080/news/articles/cde7ng85jwgo.amp?renderer_env=live
3. Confirm the media player renders and is playable
4. Visit legacy CPS page: http://localhost.bbc.com:7080/serbian/cyr/srbija-68707945.amp?renderer_env=live
5. Confirm the media player renders and is playable

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
